### DISCUSSION TITLE
Translate the user documentation for MMagic's inference and demo 翻译 MMagic 推理和 demo 的中英文使用文档

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#597] [MMagic]
# Translate the user documentation for MMagic's inference and demo 
 翻译 MMagic 推理和 demo 的中英文使用文档 

Translate [https://github.com/open-mmlab/mmagic/blob/main/docs/en/user_guides/inference.md](https://github.com/open-mmlab/mmagic/blob/main/docs/en/user_guides/inference.md)

翻译 [https://github.com/open-mmlab/mmagic/blob/main/docs/en/user_guides/inference.md](https://github.com/open-mmlab/mmagic/blob/main/docs/en/user_guides/inference.md)

## Task Information

**Repository**：MMagic

**Difficulty**：Easy

**Points**：20

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：AllYoung

**Claim Time**：2023年08月04日

**PR Link**：https://github.com/open-mmlab/mmagic/pull/1996

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by AllYoung** [comment]
https://github.com/open-mmlab/mmagic/pull/1974

**Comment by AllYoung** [comment]
https://github.com/open-mmlab/mmagic/pull/1996
